.pyup.yml

```diff
@@ -1,4 +1,7 @@
-# autogenerated pyup.io config file 
+# autogenerated pyup.io config file
 # see https://pyup.io/docs/configuration/ for all available options
```

Comment:
We can remove this comment, as it's no longer the autogenerated file.

Reply:
Done.